js/atree_constants.js

```diff
@@ -8161,15 +8176,8 @@ const atrees = {
                 {
                     "type": "add_spell_prop",
                     "base_spell": 1,
-                    "target_part": "Spin Attack",
-                    "multipliers": [
-                        0,
-                        0,
-                        0,
-                        0,
-                        20,
-                        0
-                    ]
+                    "target_part": "Per Hit",
+                    "multipliers": [0, 0, 0, 0, 20, 0]
```

Comment:
To make this work with both spin attack and lacerate, you can use `behavior: modify` and target `Per Hit` and `Spin Attack` parts